io.github.martchus.syncthingtray.yml

```
@@ -0,0 +1,147 @@
+id: io.github.martchus.syncthingtray
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: syncthingtray
+finish-args:
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.KGlobalSettings
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.kde.kconfig.notify
```

```suggestion
  - --talk-name=com.canonical.AppMenu.Registrar
```

These two are in the runtime https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/4930296f1a6d17eee3863a03c554780bb94baf62/org.kde.Sdk.json.in#L79-80